### PR TITLE
Release v1.9.0

### PR DESCRIPTION
**Pull request by lidel**
> Part of https://github.com/protocol/bifrost-infra/issues/2018 and https://github.com/ipfs/kubo/issues/9152

This release:

-  switches `ipfs-updater` to Kubo 0.15 and fetching things from  `dist.ipfs.tech` (see changes in https://github.com/ipfs/ipfs-update/pull/161, https://github.com/ipfs/ipfs-update/commit/47983018a6ecccf284da040579d5f763f0c01e04). 
   - It still uses `go-ipfs` artifact name behind the scenes to avoid unexpected problems  (Kubo does not have versions older than 0.14)
- uses version from `version.json`, removing need for manual edit of `config/config.go`

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -20,8 +21,15 @@ import (
 	"github.com/whyrusleeping/stump"
 )
 
+//go:embed  version.json
+var versionFile []byte
+
 func init() {
 	stump.ErrOut = os.Stderr
+	err := config.SetVersionNumber(versionFile)
```
Comment (Jorropo) [inline main.go:29]: This doesn't follow dependency analysis rules and break if someone use `CurrentVersionNumber` at init time.
(I know this isn't happening currently I just don't like surprises)
You should move `CurrentVersionNumber` in this package and do something like `var CurrentVersionNumber = readCurrentVersionNumberFromEmbed()`.

Comment (lidel) [reply main.go]: @Jorropo would https://github.com/ipfs/ipfs-update/pull/167/commits/8edeb6dd835deaffda8904f7fcb37637b00c5fa9 be ok? AFAIK `config.go` is 7-year-old copy-paste from go-ipfs, never used for anything.

Comment (Jorropo) [reply main.go]: perfect

**Review by Jorropo — APPROVED** [review]
LGTM

**Pull request merged**